t/lib/test-in-container-environs.sh

```diff
@@ -47,8 +47,10 @@ docker_info="$(docker info >/dev/null 2>&1)" || {
 
 docker build -t $ident.image -f $thisdir/Dockerfile.environs $thisdir
 
+docker rm -f "$containername" >&/dev/null || :
+
 map_port=""
-[ -z "$EXPOSE_PORT" ] || map_port="-p $EXPOSE_PORT:80"
+[ -z "$EXPOSE_PORT" ] || map_port="-p 80:$EXPOSE_PORT"
 docker run $map_port --rm --name "$containername" --env REBUILD=1 -d -v"$thisdir/../../..":/opt/environs/MirrorCache -- $ident.image
```

Comment:
I'd consider adding a way to somehow control MIRRORCACHE_CITY_MMDB inside container here, because default test GeoIP database in t/data/city.mmdb covers only client addresses 127.0.0.{1..4})

Reply:
E.g. MIRRORCACHE_CITY_MMDB="" may be better for testing redirection to mirror, i.e. ignoring locations completely

Reply:
So, something like this?
```bash
mc_database=""
if [[ -n "$MIRRORCACHE_CITY_MMDB" ]]; then
    mc_database="--env MIRRORCACHE_CITY_MMDB=$MIRRORCACHE_CITY_MMDB"
    [[ -f "$MIRRORCACHE_CITY_MMDB" ]] && mc_database+=" -v $MIRRORCACHE_CITY_MMDB"
fi
```
and then pass it to docker run
```bash
docker run $mc_database ....
```

Reply:
Hm. Not sure how to implement this. What I have only adds the environment variable  MIRRORCACHE_CITY_MMDB if it is set to something, not when MIRRORCACHE_CITY_MMDB=""
For example: 
```bash
$> MIRRORCACHE_CITY_MMDB=/var/lib/GeoIP/GeoLite2-City.mmdb ./01-smoke.sh
...
docker run --env MIRRORCACHE_CITY_MMDB=/var/lib/GeoIP/GeoLite2-City.mmdb -v /var/lib/GeoIP/GeoLite2-City.mmdb:/var/lib/GeoIP/GeoLite2-City.mmdb
```